tests/.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
   rules: {
     camelcase: ['error', { properties: 'never', ignoreDestructuring: true }],
     'comma-dangle': ['error', 'always-multiline'],
+    'prefer-object-spread': 'error',
```

Comment:
what's this?

Reply:
If it finds `Object.assign({}, blah)`, it will change it to `{ ...blah }`. See the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax).

Reply:
The most important difference is:

> Note that [`Object.assign()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) can be used to mutate an object, whereas spread syntax can't.